app/src/views/features/sessions/SessionViewer/DraftSessionViewer.tsx

```diff
@@ -77,11 +96,21 @@ const DraftSessionViewer: React.FC = () => {
     [dispatch]
   );
 
+  useEffect(() => {
+    if (isImportedSession && sessionRecording === null) {
+      navigate(PATHS.SESSIONS.ABSOLUTE);
+    }
+  }, [navigate, isImportedSession, sessionRecording]);
+
```

Comment:
When a user uploads a session, after successfully parsing the file it will redirect from `index` to `draft` page, now if the user refreshes the page then the draft page will be broken since we don't have the session details in the state.